.github/workflows/ci.yml

```
@@ -9,23 +9,67 @@ on:
+          python-version: '3.11'
+      - name: Install backend security scanner (Bandit)
+        run: pip install bandit
+      - name: Run backend security scan (Bandit)
+        run: bandit -r apps/api -ll
+
+      - name: Install uv
+        run: pip install uv
+
+      - name: Install API Production Dependencies for Audit
+        working-directory: ./apps/api
+        run: uv sync --frozen --no-dev # This installs dependencies based on pyproject.toml and uv.lock into the current env
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)